dart/test/sentry_tracer_test.dart

```diff
@@ -244,18 +291,17 @@ class Fixture {
   SentryTracer getSut({
     bool? sampled = true,
     bool waitForChildren = false,
+    bool trimEnd = false,
     Duration? autoFinishAfter,
   }) {
     final context = SentryTransactionContext(
       'name',
       'op',
       sampled: sampled,
     );
-    return SentryTracer(
-      context,
-      hub,
-      waitForChildren: waitForChildren,
-      autoFinishAfter: autoFinishAfter,
-    );
+    return SentryTracer(context, hub,
+        waitForChildren: waitForChildren,
+        autoFinishAfter: autoFinishAfter,
+        trimEnd: trimEnd);
```

Comment:
```suggestion
        trimEnd: trimEnd,);
```

that requires indentation again.